src/console_cmds.cpp

```diff
@@ -1877,44 +1877,120 @@ DEF_CONSOLE_CMD(ConNewGRFProfile)
 {
 	if (argc == 0) {
 		IConsoleHelp("Collect a callback profiling session from a NewGRF for a number of in-game days.");
-		IConsoleHelp("Usage: newgrf_profile [<grf-num> <num-days>]");
-		IConsoleHelp("Run without parameters to get a list of active NewGRFs and their grf-num");
+		IConsoleHelp("Usage: newgrf_profile [list]");
+		IConsoleHelp("  List all NewGRFs that can be profiled, and their status.");
+		IConsoleHelp("Usage: newgrf_profile add <grf-num>...");
```

Comment:
The "content" command uses "select" and "unselect". Does it make sense to use the same terms?

Reply:
Eh I guess that's doable. I'd rather not have long sub-command names when the base command is already long and has an underscore, but can allow abbreviations (just prefix match on "sel" and "uns"?)